.github/workflows/ci.yml

```diff
@@ -353,6 +353,10 @@ jobs:
       - name: Verify Cairo listings
         run: |
           ./scripts/verify_cairo_listings.sh
+      # TODO(#3469): Remove this step and `verify_cairo_listings_scarb_2_12.sh` script once Cairo 2.12 is the minimal version
```

Comment:
It will be after macros are merged, so we might want to wait with merging this PR.